wicket-experimental/wicket-atmosphere/pom.xml

```diff
@@ -18,13 +18,12 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<parent>
-		<groupId>org.apache.wicket.experimental.wicket7</groupId>
+		<groupId>org.apache.wicket.experimental.wicket8</groupId>
 		<artifactId>wicket-experimental</artifactId>
 		<version>8.0.0-SNAPSHOT</version>
 		<relativePath>../pom.xml</relativePath>
 	</parent>
 	<artifactId>wicket-atmosphere</artifactId>
-	<version>0.24-SNAPSHOT</version>
```

Comment:
The removal means that the version of the module will be 8.0.0-SNAPSHOT from now on.
It is intentionally 0.x to allow change in APIs because it is experimental.